content/applications/general/voip/transfer_forward.rst

```diff
@@ -0,0 +1,124 @@
+==========================================
+Make, receive, transfer, and forward calls
+==========================================
+
+Call prospective clients, customers, or colleagues is an essential part of any business. A company
+also needs to be available when the customer calls in, to build trust and make connections. This
+document covers how to make, receive, transfer, and forward calls with *VoIP*.
+
+Make calls
```

Comment:
aren't spoon feeding a bit here? I know Odoo is sometimes complex to use but explaining that the green phone icon is for answering and the red one is for hanging up seems like going a step too far in verbosity. I doubt users who struggles will that would even manage to find this documentation page.